src/AltinnCore/Common/Services/Implementation/FormSILocalDev.cs

```diff
@@ -34,7 +34,6 @@ public class FormSILocalDev : IForm
         /// <param name="repositorySettings">The service repository settings</param>
         /// <param name="httpContextAccessor">The http context accessor</param>
         /// <param name="testdataRepositorySettings">Test data repository settings</param>
```

Comment:
why is:
/// <param name="generalSettings">The general settings</param>
removed if its still a param to the method?

Reply:
I did a cleanup on warnings in the code, this will reintroduce a warning so it would be nice if you fixed it

Reply:
fixed